docs/sql/functions/nested.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: docu
+title: Nested Functions
+selected: Documentation/Functions/Nested Functions
+expanded: Functions
+---
+This section describes functions and operators for examining and manipulating nested values.
+
+| Function | Description | Example | Result |
+|:---|:---|:---|:---|
+| `list_extract(`*`list`*`, `*`index`*`)` | Extract the `index`th (0-based) value from the list. | `list_extract(l, 2)` | `6` |
```

Comment:
Maybe also mention the  [] syntax that is also supported here

Reply:
And aliases...